builds/Makefile

```diff
@@ -109,6 +109,9 @@ $(SDK_CONFIG):
 	echo "DISABLE_ANDROID=1" > $@
 	echo "DISABLE_DESKTOP=1" >> $@
 	echo "DISABLE_WASM=1" >> $@
```

Comment:
we can remove the DISABLE_ entries as they're not needed anymore (the SDKs logic was changed to only enable what you need)